MacOS builds seem to be failing for a weird reason:
```
00:09:55 CMake Warning at cmake/Modules/ECMQueryQmake.cmake:16 (message):
00:09:55   Should specify a qmake Qt5 binary.  Can't check QT_INSTALL_PREFIX
00:09:55 Call Stack (most recent call first):
00:09:55   cmake/Modules/ECMGeneratePriFile.cmake:105 (query_qmake)
00:09:55   CMakeLists.txt:18 (include)
00:09:55 
00:09:55 
00:09:55 CMake Error at /usr/local/Cellar/cmake/3.12.2/share/cmake/Modules/FindQt4.cmake:1313 (message):
00:09:55   Found unsuitable Qt version "" from NOTFOUND, this code requires Qt 4.x
00:09:55 Call Stack (most recent call first):
00:09:55   CMakeLists.txt:93 (find_package)
```

--- a/ci/Jenkinsfile.linux
+++ b/ci/Jenkinsfile.linux
@@ -31,6 +31,7 @@ pipeline {
    * https://issues.jenkins-ci.org/browse/JENKINS-49076
    **/
   environment {
+    NODE_VERSION = 'v10.14.1'
     BUILD_PLATFORM = 'linux'
     LANG = 'en_US.UTF-8'
     LANGUAGE = 'en_US.UTF-8'
@@ -44,34 +45,34 @@ pipeline {
   stages {
     stage('Prep') {
       steps {
-        script {
+        script { nvm(env.NODE_VERSION) {
           /* Necessary to load methods */
           desktop = load 'ci/desktop.groovy'
           cmn     = load 'ci/common.groovy'
           sh 'env'
           desktop.prepDeps()
-        }
+        } }
       }
     }
     stage('Lint') {
-      steps {
+      steps { nvm(env.NODE_VERSION) {
         script { cmn.runLint() }
-      }
+      } }
     }
     stage('Tests') {
-      steps {
+      steps { nvm(env.NODE_VERSION) {
         script { cmn.runTests() }
-      }
+      } }
     }
     stage('Build') {
-      steps {
+      steps { nvm(env.NODE_VERSION) {
         script { desktop.buildClojureScript() }
-      }
+      } }
     }
     stage('Compile') {
-      steps {
+      steps { nvm(env.NODE_VERSION) {
         script { desktop.compile() }
-      }
+      } }
     }
     stage('Bundle') {
       steps {

--- a/ci/Jenkinsfile.macos
+++ b/ci/Jenkinsfile.macos
@@ -14,6 +14,7 @@ pipeline {
   }
 
   environment {
+    NODE_VERSION = 'v10.14.1'
     BUILD_PLATFORM = 'macos'
     LANG = 'en_US.UTF-8'
     LANGUAGE = 'en_US.UTF-8'
@@ -25,33 +26,33 @@ pipeline {
   stages {
     stage('Prep') {
       steps {
-        script {
+        script { nvm(env.NODE_VERSION) {
           /* Necessary to load methods */
           desktop = load 'ci/desktop.groovy'
           cmn     = load 'ci/common.groovy'
           desktop.prepDeps()
-        }
+        } }
       }
     }
     stage('Lint') {
-      steps {
+      steps { nvm(env.NODE_VERSION) {
         script { cmn.runLint() }
-      }
+      } }
     }
     stage('Tests') {
-      steps {
+      steps { nvm(env.NODE_VERSION) {
         script { cmn.runTests() }
-      }
+      } }
     }
     stage('Build') {
-      steps {
+      steps { nvm(env.NODE_VERSION) {
         script { desktop.buildClojureScript() }
-      }
+      } }
     }
     stage('Compile') {
-      steps {
+      steps { nvm(env.NODE_VERSION) {
         script { desktop.compile() }
-      }
+      } }
     }
     stage('Bundle') {
       steps {

--- a/ci/Jenkinsfile.windows
+++ b/ci/Jenkinsfile.windows
@@ -30,6 +30,7 @@ pipeline {
    * https://issues.jenkins-ci.org/browse/JENKINS-49076
    **/
   environment {
+    NODE_VERSION = 'v10.14.1'
     BUILD_PLATFORM = 'linux'
     LANG = 'en_US.UTF-8'
     LANGUAGE = 'en_US.UTF-8'
@@ -45,35 +46,35 @@ pipeline {
 
   stages {
     stage('Prep') {
-      steps {
+      script { nvm(env.NODE_VERSION) {
         script {
           /* Necessary to load methods */
           desktop = load 'ci/desktop.groovy'
           cmn     = load 'ci/common.groovy'
           sh 'env'
           desktop.prepDeps()
-        }
+        } }
       }
     }
     stage('Lint') {
-      steps {
+      steps { nvm(env.NODE_VERSION) {
         script { cmn.runLint() }
-      }
+      } }
     }
     stage('Tests') {
-      steps {
+      steps { nvm(env.NODE_VERSION) {
         script { cmn.runTests() }
-      }
+      } }
     }
     stage('Build') {
-      steps {
+      steps { nvm(env.NODE_VERSION) {
         script { desktop.buildClojureScript() }
-      }
+      } }
     }
     stage('Compile') {
-      steps {
+      steps { nvm(env.NODE_VERSION) {
         script { desktop.compile() }
-      }
+      } }
     }
     stage('Bundle') {
       steps {

--- a/ci/Jenkinsfile.windows
+++ b/ci/Jenkinsfile.windows
@@ -46,7 +46,7 @@ pipeline {
 
   stages {
     stage('Prep') {
-      steps { nvm(env.NODE_VERSION) {
+      steps {
         script { nvm(env.NODE_VERSION) {
           /* Necessary to load methods */
           desktop = load 'ci/desktop.groovy'

--- a/ci/Jenkinsfile.windows
+++ b/ci/Jenkinsfile.windows
@@ -46,8 +46,8 @@ pipeline {
 
   stages {
     stage('Prep') {
-      script { nvm(env.NODE_VERSION) {
-        script {
+      steps { nvm(env.NODE_VERSION) {
+        script { nvm(env.NODE_VERSION) {
           /* Necessary to load methods */
           desktop = load 'ci/desktop.groovy'
           cmn     = load 'ci/common.groovy'

